tgcc/raw/understanding-unets/env_config.sh

```diff
@@ -5,11 +5,11 @@ module load cuda/10.1.105
 module load python3/3.7.5
 module load gnu/8.3.0
 module load mkl/19.0.5.281
-
+​
 pip install --target=$CCCWORKDIR/installed-packages/ --upgrade ./
-
-export BSD500_DATA_DIR=$CCCSCRATCHDIR/
-export BSD68_DATA_DIR=$CCCSCRATCHDIR/
-export DIV2K_DATA_DIR=$CCCSCRATCHDIR/
-export LOGS_DIR=$CCCSCRATCHDIR/
-export CHECKPOINTS_DIR=$CCCSCRATCHDIR/
+​
+export BSD500_DATA_DIR=$CCCWORKDIR/
```

Comment:
Why do you put data and the rest in work rather than scratch ?
Unless you have a good reason to do otherwise I would rather you stuck to the idris guidelines on how to use the disk spaces.

Reply:
true. Now using scratch in `learnlets.sh` file

Reply:
cool !